int32 rbl_temp
int32 rbr_temp
int32 v12_csense
int32 v12_vsense

int32 rb_rb_sig
int32 rb_lb_sig
int32 rb_da_sig

int32 wrd_sig
